### ISSUE TITLE
Custom MessageService vs. Grid Filter Clear Button Tooltip

### ISSUE THREAD

**Issue by TimoTegtmeier** [telerik/kendo-angular#1389]
## I'm submitting a...
* Bug report 

## Current behavior
I'm currently using a custom MessageService to bridge the kendo components to angular-l10n. This works almost flawless - with one exception:
When a filter has been entered for a column of a kendo-grid, the tooltip of the "Clear"-Button will always be "Clear", no matter which value I provide for "kendo.grid.filterClearButton".

## Expected behavior
The tooltip should be the string I return from the MessageService.

## Minimal reproduction of the problem with instructions
Create a message service:

```
import { Injectable } from '@angular/core';
import { MessageService } from '@progress/kendo-angular-l10n';

@Injectable()
export class KendoMessageService extends MessageService {

    get(key: string): string {
        return 'Always use this string';
    }
}
```

Wire it up in the AppModule:
```
@NgModule({
    ...
    providers: [
        ...
        { provide: MessageService, useClass: KendoMessageService }
    ]
})
```

And place a kendo-grid component on your app component.
Provide some sample data and try to filter. The tooltip of the "Clear Filter"-Button will not be 'Always use this string', but 'Clear'.

## What is the motivation or use case for changing the behavior?
I want to be able to correctly localize the tooltip.

## Environment
Package versions:
```
+-- @angular/animations@5.2.7
+-- @angular/common@5.2.7
+-- @angular/compiler@5.2.7
+-- @angular/compiler-cli@5.2.7
+-- @angular/core@5.2.7
+-- @angular/forms@5.2.7
+-- @angular/http@5.2.7
+-- @angular/platform-browser@5.2.7
+-- @angular/platform-browser-dynamic@5.2.7
+-- @angular/platform-server@5.2.7
+-- @angular/router@5.2.7
+-- @ngtools/webpack@1.10.1
+-- @progress/kendo-angular-buttons@3.0.4
+-- @progress/kendo-angular-dateinputs@2.1.1
+-- @progress/kendo-angular-dialog@3.2.0
+-- @progress/kendo-angular-dropdowns@2.0.2
+-- @progress/kendo-angular-excel-export@1.0.7
+-- @progress/kendo-angular-grid@2.1.2
+-- @progress/kendo-angular-inputs@2.2.0
+-- @progress/kendo-angular-intl@1.3.2
+-- @progress/kendo-angular-l10n@1.0.7
+-- @progress/kendo-angular-layout@2.1.1
+-- @progress/kendo-angular-popup@2.0.2
+-- @progress/kendo-data-query@1.2.0
+-- @progress/kendo-drawing@1.5.1
+-- @progress/kendo-theme-bootstrap@2.11.8
+-- @types/chai@4.1.2
+-- @types/copy-webpack-plugin@4.4.0
+-- @types/jasmine@2.8.6
+-- @types/webpack-env@1.13.5
+-- angular-l10n@4.1.5
+-- angular2-router-loader@0.3.5
+-- angular2-template-loader@0.6.2
+-- aspnet-prerendering@3.0.1
+-- aspnet-webpack@2.0.3
+-- awesome-typescript-loader@3.5.0
+-- bootstrap@4.0.0
+-- chai@4.1.2
+-- copy-webpack-plugin@4.4.1
+-- css@2.2.1
+-- css-loader@0.28.10
+-- es6-shim@0.35.3
+-- event-source-polyfill@0.0.12
+-- expose-loader@0.7.4
+-- extract-text-webpack-plugin@3.0.2
+-- file-loader@1.1.11
+-- font-awesome@4.7.0
+-- html-loader@0.5.5
+-- isomorphic-fetch@2.2.1
+-- jasmine-core@3.1.0
+-- jquery@3.3.1
+-- json-loader@0.5.7
+-- karma@2.0.0
+-- karma-chai@0.1.0
+-- karma-chrome-launcher@2.2.0
+-- karma-cli@1.0.1
+-- karma-jasmine@1.1.1
+-- karma-webpack@2.0.13
+-- nan@2.9.2 extraneous
+-- npm@5.7.1 extraneous
+-- popper.js@1.12.9
+-- preboot@6.0.0-beta.3
+-- raw-loader@0.5.1
+-- reflect-metadata@0.1.12
+-- rxjs@5.5.6
+-- style-loader@0.20.2
+-- to-string-loader@1.1.5
+-- typescript@2.7.2
+-- url-loader@1.0.1
+-- webpack@3.11.0
+-- webpack-hot-middleware@2.21.2
+-- webpack-merge@4.1.2
`-- zone.js@0.8.20

```
Browser:
- Chrome (desktop) 65.0.3325.162

System:
- TypeScript version: 2.7.2
- Node version: 8.9.4
- Platform:  Windows

**Comment by rkonstantinov** [comment]
Hey @TimoTegtmeier, this should be fixed with the 3.0.0 version of kendo-angular-grid package.

**Comment by TimoTegtmeier** [comment]
I can confirm that it now works as expected. Thank you!
